pkg/console/subresource/managedclusterview/oauth_client.go

```diff
@@ -52,16 +52,16 @@ func GetName(mcv *unstructured.Unstructured) (string, error) {
 
 func GetNamespace(mcv *unstructured.Unstructured) (string, error) {
 	namespace, found, err := unstructured.NestedString(mcv.Object, "metadata", "namespace")
-	if err != nil || !found || namespace == "" {
+	if err != nil || !found {
 		return "", err
 	}
 	return namespace, nil
 }
 
-func GetGroupVersionResource() schema.GroupVersionResource {
-	return schema.GroupVersionResource{
-		Group:    "view.open-cluster-management.io",
-		Version:  "v1beta1",
-		Resource: "managedclusterviews",
+func GetOAuthClientSecret(mcv *unstructured.Unstructured) (string, error) {
+	secret, found, err := unstructured.NestedString(mcv.Object, "status", "result", "secret")
+	if err != nil || !found {
```

Comment:
we can directly return these values
```suggestion
        return secret, err
```

Reply:
Also I cant find where this function is being used.

Reply:
https://github.com/TheRealJon/console-operator/blob/5c44b4725b3c6d3a76ffd517968d1bc6e45c00c1/pkg/console/controllers/managedclusters/controller.go#L437